components/ui/form/Schedule.tsx

```diff
@@ -73,7 +111,7 @@ const TimeRangeField = ({ name }: TimeRangeFieldProps) => {
             onFocus={() => setOptions(timeOptions())}
             onBlur={() => setOptions([])}
             defaultValue={getOption(value)}
-            onChange={(option) => onChange(new Date(option?.value as number))}
+            onChange={(option) => customOnChange(onChange, option)}
```

Comment:
@Jenietoc does it make sense to name a callback `customOnChange`? Will other devs understand what it does?